File: src/main/java/com/group310971/gormash/dto/EditorResponseTo.java
package com.group310971.gormash.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class EditorResponseTo {
    private Long id;
    private String login;
    private String firstname;
    private String lastname;
}

File: 351003/Gorodko/app/Dto/EditorResponseTo.cs
using System.Text.Json.Serialization;

namespace Project.Dto {
    public class EditorResponseTo : BaseResponseTo {
        [JsonPropertyName("login")]
        public string Login { get; set; } = string.Empty;

        [JsonPropertyName("firstname")]
        public string Firstname { get; set; } = string.Empty;

        [JsonPropertyName("lastname")]
        public string Lastname { get; set; } = string.Empty;
    }
}